subiquity/common/filesystem/tests/test_gaps.py

```diff
@@ -548,6 +563,12 @@ def test_trailing_preserved_partition(self):
             ([], 0),
             gaps.movable_trailing_partitions_and_gap_size(p1))
 
+    def test_trailing_lvm_logical_volume_no_gap(self):
```

Comment:
A test with a non-zero free space would be good I think.

Reply:
Agreed!

Reply:
updated!
I wasn't able to mock the alignment values in a way similar to disks/partitions. But I think the test looks OK.